l3build-check.lua

```diff
@@ -807,13 +807,13 @@ function runtest(name, engine, hide, ext, pdfmode, breakout)
   for _,filetype in pairs(auxfiles) do
     for _,file in pairs(filelist(testdir, filetype)) do
       if match(file,"^" .. name .. ".[^.]+$") then
-        local ext = match(file, "%.[^.]+$")
```

Comment:
ext already used, renamed to e7n

Reply:
I really don't like `e7n`, and again I'm not sure `ext` is so bad, but if it needs a new name then `exn` or `extn` or similar - I really don't find `e7n` to be clear. (I guess the 'TeX hat': looks like `e` with an argument `7` and  argument `n` to me)

Reply:
No problem, e7n means 'e' + 7 letter + 'n'.
I'll go for `extn` instead.